src/package/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.9)
+
+file(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../tool/cpp/cppwinrt/version.h cppwinrt_version_h)
+file(READ ${cppwinrt_version_h} cppwinrt_version_text)
```

Comment:
Yes, the build definition is generating version.h.  it's just not doing it thru cmake.  That seemed preferable to me, so that the baked in snapshot version of 2.0.0 would not need a build definition to create it (i.e. local dev builds would be sensible)

Reply:
Agreed. It's very helpful to have a stable version in GitHub to validate against. It should not change when building with cmake locally.

Reply:
I'm cool if we want to make that change as a seperate PR. Let me know and I'll file this an issue

Reply:
Sounds good.  If we have a way to do a local CMake build that defaults to a hardcoded baseline version (2.0.0), I'm fine with that.